src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,66 @@
+package step2;
+
+import java.util.ArrayList;
+
+public class Calculator {
+
+    ArrayList<Double> numList;
+    ArrayList<String> operationList;
```

Comment:
```suggestion
    List<Double> numbers;
    List<String> operations;
```

ArrayList와 같이 구현 클래스가 외부에 노출될 경우, 추후 변경으로 하위호환성이 깨질 우려가 있습니다.
> https://www.slipp.net/questions/345

---

List, Set 등 자료구조를 지칭하는 단어를 클래스나 변수이름으로 사용하는 것을 지양하고 있습니다.
사용하는 자료구조가 변경되면 변경의 범위가 외부로 전파될 수 있기 때문입니다.
List와 같은 복수의 데이터의 경우 복수형을 쓰는 것이 더 적합하다고 생각합니다.

https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20의미%20있는%20이름.md#그릇된-정보를-피하라

Reply:
@brainbackdoor  안녕하세요! 최수정 입니다  꼼꼼하게 리뷰 해주셔서 감사합니다. :pray:
덕분에 새로운 사실을 많이 알게되어 기쁩니다.
다름이 아니라 리뷰주신 부분 다시 변경해서 PR을 올려야되는지 여쭤보려고 메세지 남깁니다.

Reply:
아, 이런 메일 알림을 늦게 확인했네요 ㅠㅠ
merge 할 경우에는 해당 단계에서 기대했던 요구사항을 만족했다고 판단하였기 때문입니다.
그 외의 피드백은 이 후 단계에서 반영해주셔도 좋고, 이해가 가지 않거나 납득이 되지 않는 부분들은 추가적으로 질문해주시면 더 좋을 거 같습니다. 
감사합니다.